tests/rptest/tests/compatibility/arroyo_test.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2022 Redpanda Data, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+import os
+import subprocess
+
+from rptest.services.cluster import cluster
+from rptest.tests.prealloc_nodes import PreallocNodesTest
+from ducktape.cluster.remoteaccount import RemoteCommandError
+
+
+class ArroyoTest(PreallocNodesTest):
+    """
+    Run the arroyo test suite against a redpanda cluster in
+    a ducktape environment.
+
+    The test suite lives here under tests/ in https://github.com/getsentry/arroyo.
+    """
+    TEST_SUITE_PATH = "/root/external_test_suites/arroyo"
+
+    def __init__(self, ctx, *args, **kwargs):
+        super().__init__(test_context=ctx,
+                         node_prealloc_count=1,
+                         *args,
+                         **kwargs)
+
+    def _find_failed_tests(self, pytest_output: list[str]):
+        return [line for line in pytest_output if "FAILED" in line]
+
+    @cluster(num_nodes=4)
+    def test_arroyo_test_suite(self):
+        test_node = self.preallocated_nodes[0]
+
+        try:
+            env_preamble = f"DEFAULT_BROKERS={self.redpanda.brokers()}"
+            capture = test_node.account.ssh_capture(
+                f"{env_preamble} "
+                f"python3 -m pytest {ArroyoTest.TEST_SUITE_PATH} "
+                "-k KafkaStreamsTestCase -rf",
+                combine_stderr=True)
+
+            pytest_output = list(capture)
+            for log_line in pytest_output:
+                self.logger.info(log_line)
```

Comment:
```suggestion
                self.logger.debug(log_line)
```

Reply:
I think we might want info level logs here. The pytest logs are the only indication of which test in the suite failed. It doesn’t really log that much anyway.